README.md

```
@@ -56,142 +57,511 @@
-```
+---
 
-## 🤝 협업 규칙
+## 8. 추후 로드맵
 
-1. Pull Request를 통한 코드 리뷰
-2. 테스트 코드 작성 권장
-3. 문서화 필수
-4. 정기적인 코드 리뷰 
+- **꼬리질문 생성 (Follow-up Questions)**
+    - 사용자가 제출한 답변을 기반으로 3~5개의 후속 질문 자동 생성
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)